engine/termops.mli

```diff
@@ -320,3 +319,6 @@ val print_rel_context : env -> Evd.evar_map -> Pp.t
 val print_env : env -> Evd.evar_map -> Pp.t
 
 end
+
+val pr_evar_universe_context : UState.t -> Pp.t
+[@@deprecated "(8.21) Use [Evd.pr_ustate] instead"]
```

Comment:
I can't find that `pr_ustate` anywhere (currently silenced the deprecation warning in rocq-elpi)